pkg/api/v1/resource/helpers.go

```diff
@@ -100,8 +100,8 @@ func GetResourceRequest(pod *v1.Pod, resource v1.ResourceName) int64 {
 // PodResourceAllocations returns a dictionary of resources allocated to the containers of pod.
 func PodResourceAllocations(pod *v1.Pod) (allocations v1.ResourceList) {
 	allocations = v1.ResourceList{}
-	for _, container := range pod.Spec.Containers {
-		addResourceList(allocations, container.ResourcesAllocated)
+	for _, workload := range pod.Spec.Workloads() {
```

Comment:
Why is this change here? Master does not have this changes

Reply:
same here